src/components/containers/Preview.js

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import marked from 'marked';
 import styles from 'github-markdown-css/github-markdown.css';
+import { getMarkdown } from '../../selectors/document';
+import store from '../../store';
 
-function Preview({ markdown }) {
-  const __html = marked(markdown);
+function Preview() {
+  const __html = marked(getMarkdown(store.getState()));
```

Comment:
This should be a class if it interacts with redux. It also needs to subscribe to state changes.

Does your app currently work in a browser?